### PR TITLE
[10.0][FIX] stock_no_negative: show product name_get instead of name

### PR DESCRIPTION
**Pull request by Abranes**
Use `product.name_get` instead of `product.name`

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: stock_no_negative/models/stock_quant.py [hunk 1]
```diff
@@ -33,5 +33,5 @@ def check_negative_qty(self):
                     "stock level of the product '%s'%s would become negative "
                     "(%s) on the stock location '%s' and negative stock is "
                     "not allowed for this product.") % (
-                        quant.product_id.name, msg_add, quant.qty,
+                        quant.product_id.name_get()[0][1], msg_add, quant.qty,
```
Comment (rousseldenis) [inline stock_no_negative/models/stock_quant.py]:
```suggestion
                        quant.product_id.display_name, msg_add, quant.qty,
```

I think this is more correct

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@Abranes Don't forget to upgrade module version

File: stock_no_negative/models/stock_quant.py (hunk 1 above)
Comment (Abranes) [reply stock_no_negative/models/stock_quant.py]: Done!

**Review by rousseldenis — APPROVED** [review]
Fast-tracking one

**Pull request merged**